Source/SwiftLintFramework/Models/SwiftVersion.swift

```diff
@@ -23,4 +23,11 @@ enum SwiftVersion {
 
         fatalError("Unexpected Swift version")
     }()
+
+    var nameKey: String {
```

Comment:
I've created a private `nameKey(for version: SwiftVersion)` function where it's really needed. I hope it's a good enough name as it's private 😅